apps/obsidian/src/components/canvas/ExistingNodeSearch.tsx

```diff
@@ -58,10 +90,11 @@ export const ExistingNodeSearch = ({
             y: pagePoint.y - Math.random() * 100,
             props: {
               w: 200,
-              h: 100,
+              h: initialHeight,
               src,
               title: file.basename,
-              nodeTypeId: getFrontmatterForFile(plugin.app, file)?.nodeTypeId,
+              nodeTypeId: fmNodeTypeId ?? "",
+              ...(preloadedImageSrc ? { imageSrc: preloadedImageSrc } : {}),
```

Comment:
nit: Any particular reason for the spread for a single value?  Something like this would be much cleaner:
`imageSrc: preloadedImageSrc ?? "",`